Ext/Fullscreen/src/main/java/dev/testify/capture/fullscreen/FullscreenCaptureMethod.kt

```diff
@@ -65,24 +69,23 @@ import java.io.File
  */
 @Suppress("UNUSED_PARAMETER")
 fun fullscreenCapture(activity: Activity, targetView: View?): Bitmap {
-    val screenshotUtility = ScreenshotUtility()
-    activity.assureScreenshotDirectory(screenshotUtility)
+    activity.assureScreenshotDirectory()
 
     val instrumentation = InstrumentationRegistry.getInstrumentation()
     val fileName = getFileName(instrumentation.context)
-    val outputPath = OutputFileUtility().getOutputFilePath(activity, fileName)
+    val outputPath = getOutputFilePath(activity, fileName)
     val file = File(outputPath)
 
     // Use UiAutomator to take a screenshot
     val device = UiDevice.getInstance(instrumentation)
-    device.waitForIdle(1000)
+    device.waitForIdle(5000)
```

Comment:
Check: does this need to be increased?

Reply:
🤔 In theory, it should be harmless ... it's the timeout on the wait. I was having some problems with the test failing; but it wasn't due to this timeout. I changed the timeout while debugging and forgot to put it back. I could probably revert it, but I think it's overall safer to have a slightly longer timeout here. So, I think I'd like to keep it.